src/custom/utils/goerli/constants.ts

```diff
@@ -0,0 +1,26 @@
+import { Token } from '@uniswap/sdk-core'
+import { SupportedChainId as ChainId } from 'constants/chains'
+
+export const DAI_GOERLI = new Token(ChainId.GOERLI, '0x73967c6a0904aA032C103b4104747E88c566B1A2', 18, 'DAI', 'DAI')
```

Comment:
These addresses are probably not correct

Reply:
To be sure we get the right values, I'd say these have to be filled in when creating the goerli token list

Reply:
But for example DAI token should already exist on the Goerli right, we just need to find the correct address? The thing is if I go to https://goerli.etherscan.io/ and search for DAI i get a bunch of results and I'm not sure which one is the correct one.

Reply:
Yes, I expect there will be a lot of them :)
Since this is a testnet, there's no right or wrong, just ones with good liquidity.

I suggest to look for one that has liquidity on Uniswap on Goerli.

Worst case, we can deploy our own fake tokens there